rmgpy/chemkinTest.py

```diff
@@ -145,6 +149,19 @@ def testReadAndWriteTemplateReactionFamilyForMinimalExample(self):
         self.assertTrue(os.path.isfile(chemkinSavePath))
         self.assertTrue(os.path.isfile(dictionarySavePath))
 
+        # read newly written chemkin file to make sure the entire cycle works
+        _, reactions2 =loadChemkinFile(chemkinSavePath, dictionarySavePath)
```

Comment:
This line does not appear to be necessary.  Codacy points out that reactions2 is unused?

Reply:
Thanks for bringing that up. Just fixed the issue (caused by not calling reactions2 later in the method). 

Rebased. Let me know if there is anything else you find.